config/analytics_hidden_pii.yml

```diff
@@ -13,6 +13,8 @@ shared:
   :registration_interests:
     - email
   :users:
-    - trn
+    - archived_email
     - email
     - full_name
+    - preferred_name
+    - trn
```

Comment:
Can this be split to its own PR and ticket - I think we should also be doing a sync of the users table to mark existing data as hidden